lib/plugins/aws/package/compile/events/sqs/index.js

```diff
@@ -146,6 +146,11 @@ class AwsCompileSQSEvents {
               [queueLogicalId]: JSON.parse(sqsTemplate),
             };
 
+            if (event.sqs.batchSize) {
```

Comment:
Just to be sure, why do we check on `event.sqs.batchSize` here and not `event.sqs.maximumRetryAttempts` ?

If it's intentional it'll be good to add some comment clarifying such logic route

Reply:
haha nice! some good copy pasta right there. Fixed!